Emoji directive doesn't compile as documented with strict template type checking enabled

I have a project that uses:
```json
  "angularCompilerOptions": {
    "fullTemplateTypeCheck": true,
    "strictTemplates": true,
    "strictInjectionParameters": true
  }
```
I added ```emoji``` to the my templates alongside the existing markdown directives and got:
```
error TS2322: Type 'string' is not assignable to type 'boolean'.
```
I fixed this by using ```[emoji]="true"``` and everything works fine, but it's a bit messy and should at least be documented.

I believe that you should be able to fix this by changing [markdown.component.ts lines 36-38](https://github.com/jfcere/ngx-markdown/blob/master/lib/src/markdown.component.ts#L36) to read:
```typescript
  @Input()
  get emoji(): boolean | string { return this._emoji; }
  set emoji(value: boolean | string) { this._emoji = this.coerceBooleanProperty(value); }
```
and [markdown.component.ts lines 73](https://github.com/jfcere/ngx-markdown/blob/master/lib/src/markdown.component.ts#L73) to read:
```typescript
    let compiled = this.markdownService.compile(markdown, decodeHtml, this._emoji);
```

--- a/README.md
+++ b/README.md
@@ -29,7 +29,7 @@
 # ngx-markdown
 
 ngx-markdown is an [Angular](https://angular.io/) library that combines...
-- [marked](http://marked.js.org/) to parse markdown to HTML
+- [Marked](http://marked.js.org/) to parse markdown to HTML
 - [Prism.js](http://prismjs.com/) for language syntax highlight
 - [Emoji-Toolkit](https://github.com/joypixels/emoji-toolkit) for emoji support
 - [KaTeX](https://katex.org/) for math expression rendering
@@ -60,7 +60,7 @@ To add ngx-markdown library to your `package.json` use the following command.
 npm install ngx-markdown --save
 ```
 
-As the library is using [marked](https://github.com/chjj/marked) parser you will need to add `node_modules/marked/lib/marked.js` to your application.
+As the library is using [Marked](https://github.com/chjj/marked) parser you will need to add `node_modules/marked/lib/marked.js` to your application.
 
 If you are using [Angular CLI](https://cli.angular.io/) you can follow the `angular.json` example below...
 

--- a/demo/src/app/app.component.html
+++ b/demo/src/app/app.component.html
@@ -228,6 +228,10 @@
 
       <textarea class="emoji-textarea" [(ngModel)]="emojiMarkdown"></textarea>
       <markdown class="emoji-usage" [data]="emojiMarkdown" emoji></markdown>
+
+      <markdown emoji>
+        :blue_book: You can refer to this [Emoji Cheat Sheet](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md) for a complete list of _shortnames_.
+      </markdown>
     </section>
 
     <!-- PLUGINS - LineNumbers -->

--- a/lib/src/markdown.component.ts
+++ b/lib/src/markdown.component.ts
@@ -10,20 +10,25 @@ import { PrismPlugin } from './prism-plugin';
   template: '<ng-content></ng-content>',
 })
 export class MarkdownComponent implements OnChanges, AfterViewInit {
+
+  protected static ngAcceptInputType_emoji: boolean | '';
+  protected static ngAcceptInputType_katex: boolean | '';
+  protected static ngAcceptInputType_lineHighlight: boolean | '';
+  protected static ngAcceptInputType_lineNumbers: boolean | '';
+
   @Input() data: string;
   @Input() src: string;
+
+  // Plugin - emoji
+  @Input()
+  get emoji(): boolean { return this._emoji; }
+  set emoji(value: boolean) { this._emoji = this.coerceBooleanProperty(value); }
 
   // Plugin - katex
   @Input()
   get katex(): boolean { return this._katex; }
   set katex(value: boolean) { this._katex = this.coerceBooleanProperty(value); }
   @Input() katexOptions: KatexOptions;
-
-  // Plugin - lineNumbers
-  @Input()
-  get lineNumbers(): boolean { return this._lineNumbers; }
-  set lineNumbers(value: boolean) { this._lineNumbers = this.coerceBooleanProperty(value); }
-  @Input() start: number;
 
   // Plugin - lineHighlight
   @Input()
@@ -32,10 +37,11 @@ export class MarkdownComponent implements OnChanges, AfterViewInit {
   @Input() line: string | string[];
   @Input() lineOffset: number;
 
-  // Plugin - emoji
+  // Plugin - lineNumbers
   @Input()
-  get emoji(): boolean { return this._emoji; }
-  set emoji(value: boolean) { this._emoji = this.coerceBooleanProperty(value); }
+  get lineNumbers(): boolean { return this._lineNumbers; }
+  set lineNumbers(value: boolean) { this._lineNumbers = this.coerceBooleanProperty(value); }
+  @Input() start: number;
 
   // Event emitters
   @Output() error = new EventEmitter<string>();
@@ -78,7 +84,7 @@ export class MarkdownComponent implements OnChanges, AfterViewInit {
     this.ready.emit();
   }
 
-  private coerceBooleanProperty(value: boolean): boolean {
+  private coerceBooleanProperty(value: boolean | ''): boolean {
     return value != null && `${value}` !== 'false';
   }
 

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -26,6 +26,7 @@
   },
   "angularCompilerOptions": {
     "fullTemplateTypeCheck": true,
+    "strictTemplates": true,
     "strictInjectionParameters": true
   }
 }

